src/components/Picker/BasePicker/index.js

```diff
@@ -11,6 +11,26 @@ class BasePicker extends React.Component {
         super(props);
 
         this.executeOnCloseAndOnBlur = this.executeOnCloseAndOnBlur.bind(this);
+        this.setDefaultValue = this.setDefaultValue.bind(this);
```

Comment:
I don't think this is necessary, is it? It will always be in the correct scope already.